tests/e2e/suite_test.go

```diff
@@ -25,13 +25,16 @@ import (
 )
 
 const (
-	vmNameRoot         = "ros-node"
+	vmNameRoot         = "node"
 	userName           = "root"
 	userPassword       = "r0s@pwd1"
 	netDefaultFileName = "../assets/net-default.xml"
+	clusterYaml        = "../assets/cluster.yaml"
+	selectorYaml       = "../assets/selector.yaml"
 )
 
 var (
+	k3sVersion  string
```

Comment:
Nit: I would change this to kubernetesVersion as this can be any of rke, rkew or keys versions and we might want to matrix them down the line as they are a bit different.


But it's ok now.

Reply:
Yes that's true, and I already planned to add an RKE2 test, so clearly not a good variable name. What about `k8sVersion`? I found `kubernetesVersion` is a little bit too long ;-)

Reply:
I renamed the variable to `k8sVersion`. There is still hardcoded references to K3s but it's expected, as  it's for K3s installation. This will be changed as soon as I will add RKE2.